assets/src/scripts/monitoring-location/components/hydrograph/graph-brush.js

```diff
@@ -62,23 +83,38 @@ export const drawGraphBrush = function(container, store) {
         })
         .call(link(store, (svg, {layout, hydrographBrushOffset, xScale}) => {
             let selection;
+            layoutHeight = layout.height;
 
             const graphBrush = brushX()
                 .on('brush end', brushed);
 
             svg.select('.brush').remove();
 
             const group = svg.append('g').attr('class', 'brush')
-                .attr('transform', `translate(${layout.margin.left},${layout.margin.top})`);
+                .attr('transform', `translate(${layout.margin.left}, ${layout.margin.top})`);
 
+            graphBrush.handleSize([1]); // make default handle 1px wide
             graphBrush.extent([[0, 0], [layout.width - layout.margin.right, layout.height - layout.margin.bottom - layout.margin.top]]);
 
+            const brushResizePath = function(d) {
+                let e = +(d.type == 'e'),
+                    x = e ? 1 : -1,
+                    y = layoutHeight / 2;
+                return 'M' + (.5 * x) + ',' + y + 'A6,6 0 0 ' + e + ' ' + (6.5 * x) + ',' + (y + 6) + 'V' + (2 * y - 6) + 'A6,6 0 0 ' + e + ' ' + (.5 * x) + ',' + (2 * y) + 'Z' + 'M' + (2.5 * x) + ',' + (y + 8) + 'V' + (2 * y - 8) + 'M' + (4.5 * x) + ',' + (y + 8) + 'V' + (2 * y - 8);
+            };
+            customHandle = group.selectAll('.handle--custom')
+                .data([{type: 'w'}, {type: 'e'}])
+                .enter().append('path')
+                .attr('class', 'handle--custom')
+                .attr('stroke', '#000')
```

Comment:
Consider putting the stroke and cursor attributes in the style sheets

Reply:
Makes sense will do.